.idea/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Default ignored files
```

Comment:
I suggest including `.idea` in your `.gitignore` or your `.gitignore_global` files.